listenbrainz/listenstore/timescale_listenstore.py

```diff
@@ -342,29 +342,24 @@ def get_listens_query_for_dump(self, start_time, end_time):
 
         return (query, args)
 
-    def get_incremental_listens_query_batch(self, start_time, end_time, offset):
+    def get_incremental_listens_query(self, start_time, end_time):
         """
             Get a query for a batch of listens for an incremental listen dump.
             This uses the `created` column to fetch listens.
         """
 
         query = """SELECT listened_at, track_name, user_name, created, data
                      FROM listen
-                    WHERE created > :start_ts
+                    WHERE created >= :start_ts
```

Comment:
Are you sure about this? This change looks like it leads to the oldest listen to be included in two replication packets.

Reply:
Oops, makes sense, changed it.